percona-server-8.0/Dockerfile

```diff
@@ -23,6 +23,13 @@ ENV FULL_PERCONA_VERSION "$PS_VERSION.$OS_VER"
 ENV FULL_MYSQL_SHELL_VERSION "$MYSQL_SHELL_VERSION.$OS_VER"
 ENV PS_REPO release
 
+# Do not report during Docker image creation.
```

Comment:
```suggestion
# Do not report during Docker container creation.
```

Reply:
Nope, this is to not report during `docker build`

Reply:
but the initial intention of this ENV var was - disable reporting when container starts

Reply:
We have 2 cases here:
1. Docker image creation. This is `docker build ....`. During this process, package is being installed into the docker image. This process is done by us, we don't want this package installation to be reported. That's why need to disable telemetry **for the time of Docker image creation**
2. Docker container is being created. This is `docker run ...` . We want docker container creation to be reported, that's why `ps-entry.sh` calls `call-home.sh`. User can disable reporting of container creation by passing `-e PERCONA_TELEMETRY_DISABLE=1` to `docker run ...`

Reply:
ok, got, thanks